crates/xline/src/storage/index.rs

```diff
@@ -25,6 +60,14 @@ impl Index {
         }
     }
 
+    /// Creates a `IndexState`
+    pub(crate) fn state(&self) -> IndexState<'_> {
+        IndexState {
+            index_ref: self,
+            state: Mutex::default(),
+        }
+    }
+
     /// Filter out `KeyRevision` that is less than one revision and convert to `Revision`
```

Comment:
The comment seems wrong, is that greater than ?

Reply:
done